source/door_b_enb.rst

```diff
@@ -0,0 +1,174 @@
+
+FOT Procedures
+++++++++++++++
+
+* SAP 61032 (Open ACIS door) and
+* SAP 51010 (Close ACIS door) contain useful details on the door mechanism.
+
+FOT Scripts
++++++++++++
+
+* 
+
+CLD Scripts
++++++++++++
+
+* 
+
+ACIS Commands
++++++++++++++
+
+* ``1MCMDBDS`` Disable ACIS Door side B
+* ``1MCMDADS`` Disable ACIS Door side A
+
+CAPs
+++++
+
+.. |cap1438_pdf| replace:: PDF
+.. _cap1438_pdf: https://occweb.cfa.harvard.edu/occweb/FOT/configuration/CAPs/1301_1400/CAP_1438_dpaa_poweroff_recovery/CAP_1438_dpaa_poweroff_recovery.pdf
+
+.. |cap1438_doc| replace:: DOC
+.. _cap1438_doc: https://occweb.cfa.harvard.edu/occweb/FOT/configuration/CAPs/1301_1400/CAP_1438_dpaa_poweroff_recovery/CAP_1438_dpaa_poweroff_recovery.doc
+
+.. |temp_cap1438_pdf| replace:: TEMP PDF
+.. _temp_cap1438_pdf: https://occweb.cfa.harvard.edu/occweb/FOT/operations/caps_in_process/CAP_1438_ACIS_Mechanism_Disable.pdf
+
+.. |temp_cap1438_doc| replace:: TEMP DOC
+.. _temp_cap1438_doc: https://occweb.cfa.harvard.edu/occweb/FOT/operations/caps_in_process/CAP_1438_ACIS_Mechanism_Disable.doc
+
+* CAP 1438 (ACIS Mechanism Disable) (|temp_cap1438_pdf|_) (|temp_cap1438_doc|_) (link TBR: these are in the caps in process area)
+
+Relevant Notes/Memos
+--------------------
+
+* `Flight Note 394 <https://occweb.cfa.harvard.edu/occweb/FOT/configuration/flightnotes/controlled/Flight_Note394_DPA_Turn_Off_Anomaly.pdf>`_ for SEU-induced spurious PSMC commands.
+* Flight note written about this specific incident TBD
+
+A note on other similar potential anomalies
+-------------------------------------------
+
+Note that the hardware for communicating pulse commands to the PSMC is the same for
+a large number of systems, and presumably all of them are subject to SEUs which could
+be interpreted by the hardware as spurious commanding. In nearly all cases, this
+situation is benign. For example, commanding the PSMC to the existing state is a NOP.
+Commanding something to turn on which is disabled is likewise a NOP. Disabling
+a system that's active is a NOP (TBR... is this true?).
+
+There are, however, a few cases to note. Enabling a system that is normally off and
+disabled leaves us one spurious command away from activating a system inadertantly. 
+
+In the cases of the door mechanisms (side A or B) or the DEA side B, we should take
+immediate action to send a disable command, as activating the corresponding power
+supply would have negative consequences. The command to disable DEA side B is 1DEPSBDS.
+The ACIS hardware commands are documented `here <http://acis.mit.edu/acis/ipcl/ipcl-cmds-08jun98.html>`_
+(among other places).
+In the cases of the vent valve mechanisms, either the small or large vent valve,
+either side A or side B, this can be done at our leisure, since even if these
+valves were to close at this point in the mission, there would be no immediate
+consequences.
+
+See other anomaly pages for responses to spurious box turn-off commands (DPA-A, DPA-B, DEA-A).
```

Comment:
It'd be nice to make the things in parentheses links here, but it's no big deal either way.

Reply:
I'm sure there's a clever way to do that, beyond the brute-force one...?